cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -342,6 +357,22 @@ def delete(self):
             {LOWEST: 'The selected Catalog Item was deleted',
              '5.11': 'The catalog item "{}" has been successfully deleted'.format(self.name)}))
 
+    def copy(self, name=None):
+        view = navigate_to(self, 'Copy')
+
+        # there is default name like `Copy of *`
+        if name:
+            view.name.fill(name)
+
+        copied_name = view.name.value
+        view.add.click()
+        view.flash.assert_no_error()
+
+        # Catalog item can be any type
+        item_args = self.__dict__
+        item_args["name"] = copied_name
+        return self.__class__(**item_args)
```

Comment:
Q: Do we have a common way that we copy instances (with updates) in integration_tests? I have seen `from copy import copy` used, but I like this method here.

Reply:
yes... copy provide `deepcopy` and `shadow` copy but it has some limitations. when object is complex; then i use this way.